2019-Jun-09 17:12:17 log.py[line:146]/INFO/  Scrapy 1.6.0 started (bot: urlScrapy)
2019-Jun-09 17:12:17 log.py[line:149]/INFO/  Versions: lxml 4.3.3.0, libxml2 2.9.8, cssselect 1.0.3, parsel 1.5.1, w3lib 1.20.0, Twisted 19.2.0, Python 3.7.3 | packaged by conda-forge | (default, Mar 27 2019, 23:18:50) [MSC v.1900 64 bit (AMD64)], pyOpenSSL 19.0.0 (OpenSSL 1.1.1c  28 May 2019), cryptography 2.6.1, Platform Windows-10-10.0.17134-SP0
2019-Jun-09 17:12:17 crawler.py[line:38]/INFO/  Overridden settings: {'BOT_NAME': 'urlScrapy', 'CONCURRENT_REQUESTS': 10, 'CONCURRENT_REQUESTS_PER_DOMAIN': 16, 'CONCURRENT_REQUESTS_PER_IP': 16, 'DOWNLOAD_DELAY': 3, 'NEWSPIDER_MODULE': 'urlScrapy.spiders', 'ROBOTSTXT_OBEY': True, 'SPIDER_MODULES': ['urlScrapy.spiders']}
2019-Jun-09 17:12:18 telnet.py[line:60]/INFO/  Telnet Password: aadd51415e4f7637
2019-Jun-09 17:12:18 middleware.py[line:48]/INFO/  Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.logstats.LogStats']
2019-Jun-09 17:12:18 middleware.py[line:48]/INFO/  Enabled downloader middlewares:
['proxyPool.scrapy.middlewares.RetryMiddleware',
 'scrapy.downloadermiddlewares.robotstxt.RobotsTxtMiddleware',
 'proxyPool.scrapy.middlewares.ProxyMiddleware',
 'proxyPool.scrapy.middlewares.CatchExceptionMiddleware',
 'scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware',
 'scrapy.downloadermiddlewares.downloadtimeout.DownloadTimeoutMiddleware',
 'scrapy.downloadermiddlewares.defaultheaders.DefaultHeadersMiddleware',
 'proxyPool.scrapy.RandomUserAgentMiddleware.RandomUserAgentMiddleware',
 'scrapy.downloadermiddlewares.useragent.UserAgentMiddleware',
 'scrapy.downloadermiddlewares.retry.RetryMiddleware',
 'scrapy.downloadermiddlewares.redirect.MetaRefreshMiddleware',
 'scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware',
 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware',
 'scrapy.downloadermiddlewares.cookies.CookiesMiddleware',
 'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware',
 'scrapy.downloadermiddlewares.stats.DownloaderStats']
2019-Jun-09 17:12:18 middleware.py[line:48]/INFO/  Enabled spider middlewares:
['scrapy.spidermiddlewares.httperror.HttpErrorMiddleware',
 'scrapy.spidermiddlewares.offsite.OffsiteMiddleware',
 'scrapy.spidermiddlewares.referer.RefererMiddleware',
 'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware',
 'scrapy.spidermiddlewares.depth.DepthMiddleware']
2019-Jun-09 17:12:19 middleware.py[line:48]/INFO/  Enabled item pipelines:
['urlScrapy.pipelines.UrlscrapyPipeline']
2019-Jun-09 17:12:19 engine.py[line:256]/INFO/  Spider opened
2019-Jun-09 17:12:19 logstats.py[line:48]/INFO/  Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2019-Jun-09 17:12:19 telnet.py[line:74]/INFO/  Telnet console listening on 127.0.0.1:6023
2019-Jun-09 17:12:19 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://114.227.154.184:9999 】 =====
2019-Jun-09 17:12:40 retry.py[line:75]/DEBUG/  Retrying <GET http://www.alexa.cn/robots.txt> (failed 1 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 17:12:40 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://1.197.204.6:9999 】 =====
2019-Jun-09 17:13:01 retry.py[line:75]/DEBUG/  Retrying <GET http://www.alexa.cn/robots.txt> (failed 2 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 17:13:01 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://58.22.206.196:9000 】 =====
2019-Jun-09 17:13:19 logstats.py[line:48]/INFO/  Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2019-Jun-09 17:13:20 retry.py[line:91]/DEBUG/  Gave up retrying <GET http://www.alexa.cn/robots.txt> (failed 3 times): An error occurred while connecting: 10053: 你的主机中的软件中止了一个已建立的连接。.
2019-Jun-09 17:13:20 proxyDBManager.py[line:181]/DEBUG/  ===  success to update 58.22.206.196 proxy  ===
2019-Jun-09 17:13:20 proxyDBManager.py[line:181]/DEBUG/  ===  success to update 58.22.206.196 proxy  ===
2019-Jun-09 17:13:20 robotstxt.py[line:85]/ERROR/  Error downloading <GET http://www.alexa.cn/robots.txt>: An error occurred while connecting: 10053: 你的主机中的软件中止了一个已建立的连接。.
Traceback (most recent call last):
  File "E:\Users\JohnParker\Anaconda3\lib\site-packages\scrapy\core\downloader\middleware.py", line 43, in process_request
    defer.returnValue((yield download_func(request=request,spider=spider)))
twisted.internet.error.ConnectError: An error occurred while connecting: 10053: 你的主机中的软件中止了一个已建立的连接。.
2019-Jun-09 17:13:20 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://114.227.154.184:9999 】 =====
2019-Jun-09 17:13:41 retry.py[line:75]/DEBUG/  Retrying <GET http://www.alexa.cn/siterank/> (failed 1 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 17:13:41 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://1.197.16.95:9999 】 =====
2019-Jun-09 17:13:41 retry.py[line:75]/DEBUG/  Retrying <GET http://www.alexa.cn/siterank/> (failed 2 times): An error occurred while connecting: 10065: 套接字操作尝试一个无法连接的主机。.
2019-Jun-09 17:13:41 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://114.230.69.191:9999 】 =====
2019-Jun-09 17:13:44 retry.py[line:91]/DEBUG/  Gave up retrying <GET http://www.alexa.cn/siterank/> (failed 3 times): An error occurred while connecting: 10065: 套接字操作尝试一个无法连接的主机。.
2019-Jun-09 17:13:44 proxyDBManager.py[line:181]/DEBUG/  ===  success to update 114.230.69.191 proxy  ===
2019-Jun-09 17:13:44 proxyDBManager.py[line:181]/DEBUG/  ===  success to update 114.230.69.191 proxy  ===
2019-Jun-09 17:13:44 scraper.py[line:208]/ERROR/  Error downloading <GET http://www.alexa.cn/siterank/>
Traceback (most recent call last):
  File "E:\Users\JohnParker\Anaconda3\lib\site-packages\scrapy\core\downloader\middleware.py", line 43, in process_request
    defer.returnValue((yield download_func(request=request,spider=spider)))
twisted.internet.error.ConnectError: An error occurred while connecting: 10065: 套接字操作尝试一个无法连接的主机。.
2019-Jun-09 17:13:44 engine.py[line:295]/INFO/  Closing spider (finished)
2019-Jun-09 17:13:44 statscollectors.py[line:47]/INFO/  Dumping Scrapy stats:
{'downloader/exception_count': 6,
 'downloader/exception_type_count/twisted.internet.error.ConnectError': 3,
 'downloader/exception_type_count/twisted.internet.error.TCPTimedOutError': 3,
 'downloader/request_bytes': 1989,
 'downloader/request_count': 6,
 'downloader/request_method_count/GET': 6,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2019, 6, 9, 9, 13, 44, 922000),
 'log_count/DEBUG': 16,
 'log_count/ERROR': 2,
 'log_count/INFO': 10,
 'retry/count': 4,
 'retry/max_reached': 2,
 'retry/reason_count/twisted.internet.error.ConnectError': 1,
 'retry/reason_count/twisted.internet.error.TCPTimedOutError': 3,
 "robotstxt/exception_count/<class 'twisted.internet.error.ConnectError'>": 1,
 'robotstxt/request_count': 1,
 'scheduler/dequeued': 3,
 'scheduler/dequeued/memory': 3,
 'scheduler/enqueued': 3,
 'scheduler/enqueued/memory': 3,
 'start_time': datetime.datetime(2019, 6, 9, 9, 12, 19, 714482)}
2019-Jun-09 17:13:44 engine.py[line:326]/INFO/  Spider closed (finished)
